app-creator/src/widgets/ui-map/ui-map.ts

```diff
@@ -1,48 +1,143 @@
 import {} from 'googlemaps';
-import { customElement, html, LitElement } from 'lit-element';
+import { customElement, html, LitElement, property, css } from 'lit-element';
+import { InputType } from '../../redux/types/enums';
+import {
+  AttributeMetaData,
+  DefaultAttributesType,
+  getDefaultAttributes,
+} from '../../redux/types/attributes';
+
+declare global {
+  interface Window {
+    __initGoogleMap: any;
+    gm_authFailure: Function;
+  }
+}
+
+window.__initGoogleMap = window.__initGoogleMap || {};
+
+let initCalled: boolean = false;
```

Comment:
Since `initCalled` is only available within the scope of the single `ui-map` instance, it would still be undefined when we initialize a new instance. Could we instead include the script in HTML like below:
```
<script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&callback=initMap&libraries=&v=weekly" defer></script>
```
and the `google.maps.Map` namespace should be available across the entire UI? See https://developers.google.com/maps/documentation/javascript/tutorial#The_Hello_World_of_Google_Maps_v3 for a detailed example.

An alternative is to share `initCalled` with the Redux store, so it would be available across the app.